### ISSUE TITLE
No binaries are downloaded when using 'yarn add'

### ISSUE THREAD

**Issue by henrik242** [dtolstyi/node-chromium#51]
I expected this would work similarly as `npm install`, but maybe there's something I'm missing?

**Comment by dtolstyi** [comment]
To be honest, I've never tried `yarn add`. If `yarn` is executing `post install hooks` the same as `npm` does then it should work from my understanding.

**Comment by henrik242** [comment]
Hm, seems there are some conflicting answers on this topic: https://github.com/yarnpkg/yarn/issues/5476

**Comment by dtolstyi** [comment]
@henrik242 Yes, the discussion there is quite strange: for some people it works and for others it doesn't.
If you manage to find a solution, it would be great if you could make a PR to make this package working with `yarn`.
